src/pykeen/models/unimodal/mure.py

```diff
@@ -58,7 +58,8 @@ def __init__(
         # https://github.com/ibalazevic/multirelational-poincare/blob/34523a61ca7867591fd645bfb0c0807246c08660/model.py#L52
         # uses float64
         super().__init__(
-            interaction=MuREInteraction(p=p, power_norm=power_norm),
+            interaction=MuREInteraction,
+            interaction_kwargs=dict(p=p, power_norm=power_norm),
```

Comment:
We could also directly put this as default value into the `__init__` method's signature:
```python
def __init__(
  self,
  ...,
  interaction_kwargs=frozendict(p=p, power_norm=power_norm),
  ...,
)
```

Reply:
Not exactly sure if this is the same as 859db482. Why is frozendict necessary?

Reply:
Nevermind the last comment. I don't think you can actually do what you are suggesting since we're passing p and power_norm in the __init__ itself

Reply:
Yes, I was thinking about replacing it.

Before
```python
 def __init__(
        ...,
        p: int = 2,
        power_norm: bool = True,
        ...,
    ) -> None:
```

After
```python
 def __init__(
        ...,
        interaction_kwargs=frozendict(p=2, power_norm=True),
        ...,
    ) -> None:
```

The second option makes it directly clear where the parameter go to.

Reply:
Disagree. The point of the high level model class is to abstract away knowing where these parameters go or how they're handled. It should be completely flat not only for users, but also since this is how the HPO and CLI builders work